build/installers/liferay-project-sdk-with-devstudio-dxp/liferay-project-sdk-with-devstudio-dxp.xml

```diff
@@ -68,36 +68,280 @@
         <include file="../components/autodetect-java.xml" />
     </preInstallationActionList>
     <postInstallationActionList>
-        <createDirectory>
-            <path>${user_home_directory}${platform_path_separator}.liferay</path>
-            <progressText>Creating directory...</progressText>
-        </createDirectory>
         <if>
             <actionList>
-                <writeFile>
-                    <encoding>utf-8</encoding>
-                    <path>${installdir}${platform_path_separator}pwd</path>
-                    <text>${password.password}</text>
-                </writeFile>
-                <runProgram>
-                    <customErrorMessage>Could not create DXP download token. Make sure you type correct email address and password and are connected to the internet.</customErrorMessage>
-                    <program>${java_executable}</program>
-                    <programArguments>-jar "${installdir}${platform_path_separator}com.liferay.portal.tools.bundle.support.jar" createToken --force --email "${email}" --password-file "${installdir}${platform_path_separator}pwd"</programArguments>
-                    <onErrorActionList>
+                <createDirectory>
+                    <path>${user_home_directory}${platform_path_separator}.liferay</path>
+                    <progressText>Creating directory...</progressText>
+                </createDirectory>
+                <if>
+                    <actionList>
+                        <writeFile>
+                            <encoding>utf-8</encoding>
+                            <path>${installdir}${platform_path_separator}pwd</path>
+                            <text>${password.password}</text>
+                        </writeFile>
+                        <runProgram>
+                            <customErrorMessage>Could not create DXP download token. Make sure you type correct email address and password and are connected to the internet.</customErrorMessage>
+                            <program>${java_executable}</program>
+                            <programArguments>-jar "${installdir}${platform_path_separator}com.liferay.portal.tools.bundle.support.jar" createToken --force --email "${email}" --password-file "${installdir}${platform_path_separator}pwd"</programArguments>
```

Comment:
if there is proxyHost and proxy details you must pass them into this command.  `java -DproxyHost=.... -jar...`